src/dlload.c

```diff
@@ -63,10 +63,14 @@ DLLEXPORT int jl_uv_dlopen(const char *filename, jl_uv_libhandle lib_, unsigned
 #endif
                          );
     if (lib->handle) {
+        if (lib->errmsg)
+            free(lib->errmsg);
```

Comment:
It is [deliberately set to NULL here](https://github.com/JuliaLang/julia/blob/c722b5d042ae098028af6efe1659320f88f5b2f6/src/dlload.c#L89).